corehq/apps/smsbillables/tasks.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from datetime import date
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.conf import settings
+
+from corehq.apps.smsbillables.models import (
+    SmsBillable,
+    SmsGatewayFeeCriteria,
+)
+from corehq.util.log import send_HTML_email
+from dimagi.utils.dates import add_months_to_date
+
+
+@periodic_task(run_every=crontab(day_of_month='1', hour=5, minute=0), queue='background_queue', acks_late=True)
+def send_gateway_fee_report_out():
+    backend_api_ids = SmsGatewayFeeCriteria.objects.values_list('backend_api_id', flat=True).distinct()
+    first_day_previous_month = add_months_to_date(date.today(), -1)
```

Comment:
using `date.today()` might cause issues in some environments because celery tasks can be configured to be scheduled in a different timezone than what might be the local timezone of the machine (which is what `date.today()` will use).

as a precaution to avoid these complications you may want to run this on the second day of the month, or include the year and month as part of the email so there's no chance for confusion.

Reply:
That's an interesting observation, which could have implications for invoicing as well.

What timezone does ```SMS.date``` use?  (I'm concerned that if invoices for May 1 are sent while SMS's from April 30 are still being sent, then we may not count them towards invoicing.)

Reply:
`SMS.date` always uses utc

Reply:
I moved the task to 1pm, which for all timezones is guaranteed to be the new month in UTC.